### PR TITLE
Rearrange tests

### PR DESCRIPTION
**Pull request by diegoamc**
Create a separate file for different classes to improve readability.

### CODE REVIEW

**Comment by diegoamc** [comment]
Maybe we should do the same for the factories, what do you think?

File: tests/configurations/test_reading.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+    def test_save_prefix(self):
+        assert_equal(self.subject.save_prefix(), "reading_groups/{}".format(self.subject.reading_group_id))
+
+    def test_update_prefix(self):
+        assert_equal(self.subject.update_prefix(), "reading_groups/{}".format(self.subject.reading_group_id))
+
+    def test_delete_prefix(self):
+        assert_equal(self.subject.delete_prefix(), "reading_groups/{}".format(self.subject.reading_group_id))
+
```
Comment (rafamanzo) [inline tests/configurations/test_reading.py]: Looks like there are 2 extra newlines here.

File: tests/configurations/test_reading_group.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+    def test_readings(self):
+        readings_hash = {"readings": [self.reading._asdict()]}
+        with patch.object(self.subject, 'request', return_value=readings_hash) as request_mock, \
+             patch.object(Reading, 'response_to_objects_array', return_value=self.readings) as mock:
+            self.subject.readings()
+            request_mock.assert_called_once_with(":id/readings", {'id': 1}, method='get')
+            mock.assert_called_once_with(readings_hash)
+
+
```
Comment (rafamanzo) [inline tests/configurations/test_reading_group.py]: Looks like there is one extra newline here.

**Comment by rafamanzo** [comment]
Nice work with `Statistics` and the whole refactor :smile: 

It's interesting how some test files ramain long. This may be a hint that the classes they are testing may have more than one responsability.

**Pull request merged**